tests/sanity/ignore.txt

```diff
@@ -550,9 +550,11 @@ test/units/utils/test_shlex.py future-import-boilerplate
 test/units/utils/test_shlex.py metaclass-boilerplate
 test/utils/shippable/check_matrix.py replace-urlopen
 test/utils/shippable/timing.py shebang
-plugins/modules/nios_a_record.py:0:0: doc-required-mismatch: Argument 'host' in argument_spec found in provider is not required, but is documented as being required
-plugins/modules/nios_a_record.py:0:0: doc-required-mismatch: Argument 'ipv4addr' in argument_spec is not required, but is documented as being required
-plugins/modules/nios_a_record.py:0:0: doc-required-mismatch: Argument 'view' in argument_spec is not required, but is documented as being required
-plugins/modules/nios_a_record.py:0:0: invalid-ansiblemodule-schema: AnsibleModule.argument_spec.ipv4addr.ib_req: extra keys not allowed @ data['argument_spec']['ipv4addr']['ib_req']. Got True
-plugins/modules/nios_a_record.py:0:0: invalid-ansiblemodule-schema: AnsibleModule.argument_spec.name.ib_req: extra keys not allowed @ data['argument_spec']['name']['ib_req']. Got True
+plugins/modules/nios_a_record.py:0:0: doc-required-mismatch
```

Comment:
@Vaishnavi-infoblox These lines should be part of ignore-2.10.txt file and not ignore.txt file

Reply:
@Vaishnavi-infoblox The location of these files needs to be inside the collection structure which in your case should be `ansible_collections/infoblox/nios_modules/tests/sanity/ignore-2.10.txt` or `ansible_collections/infoblox/nios_modules/tests/sanity/ignore-2.9.txt`

I also see a lot of unrequired lines in your sanity files please remove all of those and start with a basic lines as below:
```
plugins/modules/nios_a_record.py validate-modules:doc-required-mismatch
plugins/modules/nios_aaaa_record.py  validate-modules:doc-required-mismatch
plugins/modules/nios_a_record.py  validate-modules:module-invalid-version-added
plugins/modules/nios_a_record.py  validate-modules:invalid-ansiblemodule-schema
plugins/modules/nios_aaaa_record.py  validate-modules:doc-required-mismatch
plugins/modules/nios_aaaa_record.py  validate-modules:invalid-ansiblemodule-schema
plugins/modules/nios_aaaa_record.py  validate-modules:module-invalid-version-added
plugins/modules/nios_zone.py validate-modules:doc-required-mismatch
plugins/modules/nios_fixed_address.py validate-modules:doc-missing-type
plugins/modules/nios_fixed_address.py validate-modules:invalid-ansiblemodule-schema
```
for the 2.10 ignore file. There is a slight change in schema that I missed last time
please check if this helps